### PR TITLE
Presentation hints

### PR DESCRIPTION
**Pull request by HadrienGardeur**
Reorganized the specification and related documents to move away from a `rendition` object defined in an EPUB extension to a new Presentation Hints document.

TODO: define new properties that are introduced by DiViNa (`fit`, `viewportRatio` and `continuous`)

### CODE REVIEW

**Comment by danielweck** [comment]
Let's enjoy an old spreadsheet:
https://docs.google.com/spreadsheets/d/1vzylT0YKP8ITftALNzuAk_fucdZ4eipu0vsnVf-8UFk/
:)

**Comment by chocolatkey** [comment]
@HadrienGardeur What about the "fit" key for presentation? I see it referenced in multiple code snippets, but it does not have its own entry here https://github.com/readium/webpub-manifest/blob/presentation-hints/extensions/presentation.md#1-presentation-hints-in-metadata

**Comment by HadrienGardeur** [comment]
@chocolatkey I've added `fit` to the document, but this is still a WIP. I've focused on the definitions, schema and examples first and will flesh things out gradually over time.

**Comment by llemeurfr** [comment]
doesn't `fit` belong to the DiVINa specific properties? it seems to have nothing to do in generic properties, also used by ebooks and audiobooks.

**Comment by HadrienGardeur** [comment]
`fit` could apply to video or HTML as well. In fact, the default behavior for a FXL EPUB is a `fit` set to `contain`.

The one that feels IMO a little bit too specific is `viewportRatio`. It's also much more underdefined than the other ones, with a clear lack of examples even in the DiViNa working docs.

**Comment by HadrienGardeur** [comment]
I'm ready to merge this PR now, `viewportRatio` can be defined later.

File: extensions/divina.md [hunk 1]
```diff
@@ -192,53 +182,64 @@ As an alternative, the manifest can also be added to a CBZ file at the same well
...
-With the first approach, the `readingOrder` contains a single image:
 
 ```
+"metadata": {
+  "title": "Manga",
+  "identifier": "https://example.com/manga",
+  "readingProgression": "rtl",
+  "presentation": {
+    "fit": "contain",
+    "spread": "landscape"
```
Comment (llemeurfr) [inline extensions/divina.md]: manga production is mostly in portrait mode.

File: extensions/divina.md [hunk 2]
```diff
@@ -150,28 +142,26 @@ This document introduces a new collection role to fulfill that goal:
 | ----- | ---------- | ------------------- | --------- |
 | `guided` | Identifies a collection containing guided navigation into a publication. | Yes  | No  |
 
-To avoid duplicating content between `readingOrder` and `guided`, Link Objects referenced in `guided` <span class="rfc">should</span> only contain `href`, `title` and `children`.
+To avoid duplicating content between `readingOrder` and `guided`, Link Objects referenced in `guided` <span class="rfc">must</span> only contain `href` and `title`.
 
 This current draft does not cover guided navigation over alternate versions of each image resource.
```
Comment (llemeurfr) [inline extensions/divina.md]: we can now say "specification" isn't it?

File: extensions/divina.md (hunk 1 above)
Comment (HadrienGardeur) [reply extensions/divina.md]: This is for spreads, which IMO is correct = display spreads only in landscape mode.

File: extensions/divina.md (hunk 2 above)
Comment (HadrienGardeur) [reply extensions/divina.md]: It's still a draft, we haven't covered everything planned in there yet.

File: extensions/divina.md (hunk 1 above)
Comment (llemeurfr) [reply extensions/divina.md]: you're right.

**Comment by chocolatkey** [comment]
`"description": "ndicates how the progression between resources` ndicates -> Indicates

**Pull request merged**